docs/developers/tutorial.md

```diff
@@ -181,7 +181,7 @@ To fix this bug, we first match function call nodes `Term.Apply` and pattern
 match only `Lit.Boolean` that appear in argument position
 
 ```scala
-case Term.Apply(_, args) =>
+case Term.Apply.After_4_6_0(_, args) =>
```

Comment:
git history for https://github.com/scalacenter/scalafix-named-literal-arguments should be rewritten